vcbuild.bat

```diff
@@ -14,10 +14,11 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=
 set target=Build
+set target_arch=ia32
+set target_env=vs2015
```

Comment:
I simply removed the default value (the downstream code test works as expected with an empty or undefined value anyway)